src/components/shared/Forms/Buttons/Buttons.tsx

```diff
@@ -40,6 +40,20 @@ export const OutlineBtn = (props: any) => {
   );
 };
 
+export const OutlineBtn2 = (props: any) => {
```

Comment:
I think we can manage with just one Outline Button style.  If needed, we could use a prop to conditionally adjust the styling instead of adding another variant.